src/util.cpp

```diff
@@ -682,7 +682,36 @@ boost::filesystem::path GetTempPath() {
 
 void runCommand(std::string strCommand)
 {
-    int nErr = ::system(strCommand.c_str());
+    int nErr = 0;
+#ifdef WIN32
+    // If we're in Windows, try using CreateProcess first.
+    ULONG nExitCode;
+    STARTUPINFO info={sizeof(info)};
+    ZeroMemory(&info, sizeof(STARTUPINFO));
+    info.cb = sizeof(STARTUPINFO);
+    info.dwFlags = STARTF_USESHOWWINDOW;
+    info.wShowWindow = SW_HIDE;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&processInfo, sizeof(PROCESS_INFORMATION));
+    if (CreateProcess(NULL, &strCommand[0], NULL, NULL, false, 0, NULL, NULL, &info, &processInfo))
```

Comment:
please use `strCommand.c_str()` (if you need zero-terminated) or `strCommand.data()` (if no zero terminated is necessary) i.s.o `&strCommand[0]`